src/screens/choice-exercises/ArticleChoiceExerciseScreen.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react'
```

Comment:
Should we still keep the `choice-exercises` directory @SteffiStuffel as there are no services anymore and two screens using the same components?

Reply:
Yes please keep the `choice-exercises` directory, as it fits into the planned (future) structure.